frontend/views/containers/chatroom/MessageBase.vue

```diff
@@ -200,20 +200,22 @@ export default ({
           }
         ]
       }
-      const possibleMentions = [
-        ...Object.keys(this.chatRoomUsers).map(u => makeMentionFromUsername(u).me),
-        makeMentionFromUsername('').all
-      ]
+      const allMention = makeMentionFromUserID('').all
+      const possibleMentions = Object.keys(this.chatRoomMembers).map(u => makeMentionFromUserID(u).me).filter(v => !!v)
 
       return text
-        .split(new RegExp(`(${possibleMentions.join('|')})`))
-        .map(t => possibleMentions.includes(t)
-          ? { type: TextObjectType.Mention, text: t }
-          : {
-              type: TextObjectType.Text,
-              text: this.convertTextToMarkdown ? convertToMarkdown(t) : t
-            }
-        )
+        .split(new RegExp(`(?<=\\s|^)(${allMention}|${possibleMentions.join('|')})(?=[^\\w\\d]|$)`))
```

Comment:
What are `(?<=\\s|^)` and `(?=[^\\w\\d]|$)`? Can you add a comment explaining this regex?

Reply:
For some reason GitHub doesn't show that this has been updated